### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -85,6 +85,7 @@ abosulte->absolute
 abosultely->absolutely
 abosulute->absolute
 abosulutely->absolutely
+abot->about
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:88]: Please suggest these additional words
```text
abot->about, abort, a bot,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11275,6 +11282,7 @@ dispertion->dispersion
 dispicable->despicable
 dispite->despite
 displa->display
+displacemnt->displacement
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:11285]:
```text
displacemnts->displacements
```
https://github.com/search?q=displacemnts&type=code

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -12255,6 +12263,9 @@ einstance->instance
 eisntance->instance
 eiter->either
 eith->with
+elaspe->elapse
+elasped->elapsed
+elasping->elapsing
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:12268]:
```text
elaspes->elapses
```
https://github.com/search?q=elaspes&type=code

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -18189,6 +18200,8 @@ inpacted->impacted
 inpacting->impacting
 inpacts->impacts
 inpeach->impeach
+inpection->inspection
+inpections->inspections
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:18204]:
```text
inpecting->inspecting
```
https://github.com/search?q=inpecting&type=code

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -22773,6 +22788,7 @@ optoin->option
 optoins->options
 optomism->optimism
 opton->option
+optonal->optional
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:22791]:
```text
optonally->optionally
```
https://github.com/search?q=optonally&type=code

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -23588,8 +23604,12 @@ peculure->peculiar
 pedestrain->pedestrian
 peding->pending
 pedning->pending
+pefer->prefer
+pefered->preferred
 peference->preference
 peferences->preferences
+peferred->preferred
+pefers->prefers
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:23612]:
```text
peferable->preferable
peferably->preferably
peferential->preferential
peferentially->preferentially
peferring->preferring
```
https://github.com/search?q=peferable&type=code
https://github.com/search?q=peferably&type=code
https://github.com/search?q=peferential&type=code
https://github.com/search?q=peferentially&type=code
https://github.com/search?q=peferring&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:23621]: [bc1a996](https://github.com/codespell-project/codespell/pull/2432/commits/bc1a996df9a29bccdfa7e8a4964a56aed45c225a)

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:22794]: [bc1a996](https://github.com/codespell-project/codespell/pull/2432/commits/bc1a996df9a29bccdfa7e8a4964a56aed45c225a)

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:18207]: [bc1a996](https://github.com/codespell-project/codespell/pull/2432/commits/bc1a996df9a29bccdfa7e8a4964a56aed45c225a)

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:12270]: [bc1a996](https://github.com/codespell-project/codespell/pull/2432/commits/bc1a996df9a29bccdfa7e8a4964a56aed45c225a)

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:11285]: [bc1a996](https://github.com/codespell-project/codespell/pull/2432/commits/bc1a996df9a29bccdfa7e8a4964a56aed45c225a)

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: [bc1a996](https://github.com/codespell-project/codespell/pull/2432/commits/bc1a996df9a29bccdfa7e8a4964a56aed45c225a)

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**